### ISSUE TITLE
I want to get the output message of the command, what should I do?

### ISSUE THREAD

**Issue by 344089386** [sshnet/SSH.NET#1139]
```
using (var shell = sshclient.CreateShellStream("", 0, 0, 0, 0, 0))
{
    var result = SendCommand(shell, "vi /tmp/saimeite/111.txt");
    Console.WriteLine(result);
    SendCommand(shell, "i");
    Console.WriteLine(result);
    SendCommand(shell, "333");
    Console.WriteLine(result);
    SendCommand(shell, "\x1b"); //Esc
    Console.WriteLine(result);
    SendCommand(shell, ":wq!");
    Console.WriteLine(result);
}

public static string SendCommand(ShellStream shellStream, string command)
{
    StringBuilder result = new StringBuilder();

    Console.WriteLine("[SSH]:" + command);

    var reader = new StreamReader(shellStream);
    var writer = new StreamWriter(shellStream);
    writer.WriteLine(command);
    writer.Flush();

    long outputStreamLength = 0;
    while (!shellStream.DataAvailable || shellStream.Length != outputStreamLength)
    {
        outputStreamLength = shellStream.Length;
        Thread.Sleep(500);
    }

    while (reader.Peek() != -1)
    {
        result.AppendLine(reader.ReadLine());
    }

    Console.WriteLine("[SSH Result]\n" + result);
    return result.ToString();
}
```

In fact, there are times when the return value of SendCommand is incorrect.

**Comment by 344089386** [comment]
I actually want to execute several commands in order, but only after the previous commands have finished

**Comment by WojciechNagorski** [comment]
SSH.NET version 2024.0.0 has changed a lot. I'm closing the ticket, but if the problem still persists, please feel free to reopen or open a new ticket.
